File: maria018/programação_basica/questão13.py
x0 = float(input(" Digite a coordenada X do primeiro ponto: "))
y0 = float(input(" Digite a coordenada Y do primeiro ponto: "))
x1 = float(input(" Digite a coordenada X do segundo ponto: "))
y1 = float(input(" Digite a coordenada Y do segudo ponto: "))

d = ((x1 - x0) * (x1 - x0) + (y1 - y0) * (y1 - y0)) ** 0.5

print(d)

File: maria018/programação_basica/Questão13.java
import java.util.Scanner;
public class Questão13 {
    public static void main (String[] args){
        Scanner s;
        float x0;
        float x1;
        float y0;
        float y1;
        float d;
        s = new Scanner(System.in);

        System.out.print(" Digite a coodernada X do primeiro ponto: ");
        x0 = s.nextFloat();
        System.out.print(" Digite a coodernada Y do primerio ponto: ");
        y0 = s.nextFloat();
        System.out.print(" Digite a coordenada X do segundo ponto: ");
        x1 = s.nextFloat();
        System.out.print(" Digite a coordenada Y do segundo ponto: ");
        y1 = s.nextFloat();


        double soma = Math.sqrt((x1 - x0) * (x1 - x0) + (y1 - y0) * (y1 - y0)); 
        System.out.println(soma);

    }
}
